### PR TITLE
[sui-studio] Add testing to the dev mode

### PR DESCRIPTION
**Pull request by carlosvillu**
This change is [Reviewable](https://reviewable.io/reviews/sui-components/sui/691)

### CODE REVIEW

File: packages/sui-studio/src/index.html [hunk 1]
```diff
@@ -7,9 +7,12 @@
   <link rel='preload' href='https://cdnjs.cloudflare.com/ajax/libs/babel-polyfill/6.26.0/polyfill.min.js' as='script'>
 
   <link id='async-css' rel="async-stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/github-markdown-css/2.4.1/github-markdown.min.css">
+  <link rel="stylesheet" href="https://unpkg.com/mocha/mocha.css" />
 
   <script src='https://cdnjs.cloudflare.com/ajax/libs/babel-polyfill/6.26.0/polyfill.min.js'></script>
   <script src='https://unpkg.com/@babel/standalone@7.0.0-beta.38/babel.min.js'></script>
+  <script src="https://unpkg.com/chai/chai.js"></script>
```
Comment (carlosvillu) [inline packages/sui-studio/src/index.html]: Remove it!

File: packages/sui-studio/workbench/src/index.html [hunk 2]
```diff
@@ -7,9 +7,16 @@
   <link rel='preload' href='https://cdnjs.cloudflare.com/ajax/libs/babel-polyfill/6.26.0/polyfill.min.js' as='script'>
 
   <link id='async-css' rel="async-stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/github-markdown-css/2.4.1/github-markdown.min.css">
+  <link rel="stylesheet" href="https://unpkg.com/mocha/mocha.css" />
 
   <script src='https://cdnjs.cloudflare.com/ajax/libs/babel-polyfill/6.26.0/polyfill.min.js'></script>
   <script src='https://unpkg.com/@babel/standalone@7.0.0-beta.38/babel.min.js'></script>
+  <script src="https://unpkg.com/chai/chai.js"></script>
```
Comment (carlosvillu) [inline packages/sui-studio/workbench/src/index.html]: Remove it!

File: packages/sui-studio/package.json [hunk 3]
```diff
@@ -30,7 +30,10 @@
     "@s-ui/helpers": "1",
     "@s-ui/mono": "1",
     "@s-ui/react-context": "1",
+    "@testing-library/react": "9.3.0",
```
Comment (tureey) [inline packages/sui-studio/package.json]: :clap:

File: packages/sui-studio/README.md [hunk 4]
```diff
@@ -62,35 +60,108 @@ Add the script to your package.json
...
+### `$ sui-studio dev`
+
+Launch a development environment where you can work in total isolation on your component.
+
+## Testing
 
-Executes internally `$ sui-mono link`, that links all components to each other.
+Test the studio's components both in the demo as in the development environment. *Currently in experimental mode*
 
+All previous commands suppor the `--experimental-test` flag, which is mandatory to see the tests in the studio's interface.
```
Comment (jcger) [inline packages/sui-studio/README.md]: typo suppor

File: packages/sui-studio/README.md [hunk 5]
```diff
@@ -62,35 +60,108 @@ Add the script to your package.json
...
 ```
 
 ## CLI
 
-### `$ sui-studio link-all`
+### `$ sui-studio start`
+
+Launch a development environment where you can see all your components at once. If there are too many components, use the `dev`command. 
+
+### `sui-studio build`
```
Comment (midudev) [inline packages/sui-studio/README.md]: Missing `$` before the command to follow the same pattern as the others.

**Pull request merged**